docs/installation.rst

```diff
@@ -0,0 +1,85 @@
+Installation
+============
+
+Prerequisites
+-------------
+
+* Python 2.7.x
+
+* pip_ > 7.x
+
+* Toil 3.3.0 or later
+
+.. _pip: https://pip.readthedocs.org/en/latest/installing.html
+
+Basic installation
+------------------
+
+To set up a basic toil-lib installation use
+
+::
+
+    pip install toil-lib
+
+
+Building & testing
+------------------
+
+For developers and people interested in building the project from source the
+following explains how to setup virtualenv to create an environment to use toil-lib
```

Comment:
Instead of this paragraph, I would say that our make flow requires a virtualenv to be installed, why that is, and then how to install virtualenv. `virtualenv` doesn't get installed by default with python, so I wouldn't assume that people know about it.